pkg/chunk/disk_cache.go

```diff
@@ -693,6 +694,10 @@ func newCacheManager(config *Config, reg prometheus.Registerer, uploader func(ke
 			Name: "staging_block_bytes",
 			Help: "Total bytes of blocks in the staging path.",
 		}),
+		stageBlockDelay: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "staging_block_delay_seconds",
+			Help: "Total seconds of delay for staging blocks",
+		}),
```

Comment:
Should this metric be a histogram?

Reply:
A histogram can provide more information with more metrics, but a counter may be enough here (less metrics).